core/templates/pages/exploration-player-page/services/player-position.service.ts

```diff
@@ -23,7 +23,7 @@ import { PlayerTranscriptService } from
   'pages/exploration-player-page/services/player-transcript.service';
 import { StateCard } from 'domain/state_card/state-card.model';
 
-interface HelpCardEventResponse {
+export interface HelpCardEventResponse {
```

Comment:
Did we add this `export` just for convenience?

Reply:
Used in `supplemental-card.component.spec.ts` L99.